### DISCUSSION TITLE
Thank You ❤️

### DISCUSSION THREAD

**Discussion by maltesa** [sannajammeh/tw-classed#149] [General]
Honestly, every time I face an issue with composing components, I look into the documentation and there is a solution. In my opinion, it is as perfect a solution for building composable base UI-Components as possible, and it should be way more popular. You deserve attention from well known react youtubers etc. to boost this libs popularity!

**Thank you for your work!**

**Comment by sannajammeh** [comment]
Love the appreciation! <3
